src/harness/sas.py

```diff
@@ -203,6 +208,42 @@ def _GetDefaultSasSSLCertPath(self):
   def _GetDefaultSasSSLKeyPath(self):
     return os.path.join('certs', 'client.key')
 
+  def DownloadFile(self, url, ssl_cert=None, ssl_key=None):
+    return self._DownloadFile('%s' % url,
+                 ssl_cert if ssl_cert else self._GetDefaultSasSSLCertPath(),
+                 ssl_key if ssl_key else self._GetDefaultSasSSLKeyPath())
+
+  def _DownloadFile(self, url, ssl_cert, ssl_key):
```

Comment:
What is this function supposed to achieve that _RequestGet does not already do? It appears to be the same function but with some values hard-coded. This includes the ciphers which is something the security tests need to modify (hence the use of TlsConfig)

Reply:
This   portion of code has been removed from this PR as Google will take over the   implementation of the FAD Object.